JPS_Ontology/ontology/ontodevice/OntoDevice.owl

```diff
@@ -131,7 +155,7 @@
     <owl:ObjectProperty rdf:about="https://www.theworldavatar.com/kg/ontodevice/hasDefaultPosition">
         <rdfs:domain rdf:resource="https://www.theworldavatar.com/kg/ontodevice/Valve"/>
         <rdfs:range rdf:resource="https://www.theworldavatar.com/kg/ontodevice/DefaultValvePosition"/>
-        <rdfs:comment>A relationship that describes a valve and it's default position (NC, NO etc).</rdfs:comment>
+        <rdfs:comment>A relationship that describes a valve and it&apos;s default position (NC, NO etc).</rdfs:comment>
```

Comment:
The characters &apos; needs to be manually replaced with '

Simply open up the owl file on notepad and go to "Edit" -> "Replace" .